src/components/Handlers/Handlers.js

```diff
@@ -2,32 +2,23 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { withModules } from '../Modules';
 import { stripesShape } from '../../Stripes';
+import withHandlers from './withHandlers';
```

Comment:
I guess I'm a little uneasy about this being called `withHandlers` when it doesn't put handlers into the props. It ought really be have a clumsy name like `withGetComponentsFromHandlers` since that's what it provides. I can see why you didn't do that, but in the long run we might be better served by explicitness than by terseness, despite our justified fear of ending up with Java-style names like `PutFixedDueDateScheduleStorageFixedDueDateSchedulesByFixedDueDateScheduleIdResponse`.

Reply:
@MikeTaylor exactly perhaps HOC is not needed now when I look at it again this morning. Also 100% agree about these long names. Do you have any suggestions for a better name?